tdrs-frontend/package.json

```diff
@@ -28,8 +28,9 @@
     "uuid": "^8.3.2"
   },
   "resolutions": {
-    "lodash": "^4.17.20",
     "bl": "^4.0.3",
+    "coa": "2.0.2",
```

Comment:
Nit, but when we add resolutions like this, we also need to run `yarn` to get an updated `yarn.lock`

Reply:
It actually results in the same yarn.lock:
```
❯ yarn                                                                                                                                                                                                                                                                ─╯
yarn install v1.22.15
[1/4] Resolving packages...
warning Resolution field "ssri@8.0.1" is incompatible with requested version "ssri@^6.0.1"
success Already up-to-date.
Done in 1.09s.
```